app/features/accounts/account-selector.tsx

```diff
@@ -83,7 +84,9 @@ export function AccountSelector<T extends Account>({
       </DrawerTrigger>
       <DrawerContent className="font-primary">
         <div className="mx-auto w-full max-w-sm">
-          <DrawerHeader>Select account</DrawerHeader>
+          <DrawerHeader>
+            <DrawerTitle>Select account</DrawerTitle>
```

Comment:
added this to remove annoying a11y error from console

Reply:
we still have a warning that DrawerDescription is missing that is a bit annoying (same in currency selector). perhaps we could add some description here but if we really don't want to I guess we can also just ignore it